db/schema.rb

```diff
@@ -1,4 +1,3 @@
-# encoding: UTF-8
```

Comment:
This... I think happened because I'm working on (gasp) a windows machine. I had a feeling I shouldn't have committed it :crying_cat_face: 

I'll revert.